packages/framework-common-helpers/src/instances.ts

```diff
@@ -0,0 +1,15 @@
+import { Class } from '@boostercloud/framework-types'
+
+export function createInstance<T>(instanceClass: Class<T>, rawObject: Record<string, any>): T {
+  const instance = new instanceClass()
+  Object.assign(instance, rawObject)
+  return instance
+}
+
+export function createInstances<T>(instanceClass: Class<T>, rawObjects: Array<Record<string, any>>): T[] {
+  return rawObjects.map((rawObject) => {
+    const instance = new instanceClass()
+    Object.assign(instance, rawObject)
+    return instance
+  })
```

Comment:
What about reusing the above method?
```suggestion
  return rawObjects.map((rawObject) => createInstance(instanceClass, rawObject))
```